packages/web/env-config.js.template

```diff
@@ -9,7 +9,7 @@ const environments = {
       projectId: 'celo-faucet',
     },
     LEADERBOARD: {
-      uri: "http://localhost:8080/v1/graphql",
+      uri: "http://localhost:4000/graphiql",
```

Comment:
I assume eventually this will be a non localhost URL?

Reply:
yes, it should be the one of `blockscout`